app/helpers/application_helper/toolbar/middleware_servers_center.rb

```diff
@@ -46,7 +46,28 @@ class ApplicationHelper::Toolbar::MiddlewareServersCenter < ApplicationHelper::T
           N_('Edit Tags'),
           :url_parms => "main_div",
           :enabled   => false,
-          :onwhen    => "1+"),
+          :onwhen    => "1+")
+      ]
+    ),
+  ])
+  button_group('middleware_server_operations', [
+    select(
+      :middleware_server_power_choice,
+      'fa fa-power-off fa-lg',
+      t = N_('Power'),
+      t,
+      :enabled => false,
+      :onwhen  => "1+",
+      :items   => [
+        button(
+          :middleware_server_reload,
+          'pficon pficon-restart fa-lg',
+          N_('Reload this #{ui_lookup(:table=>"middleware_server")}'),
+          N_('Reload Server'),
+          :url_parms => "main_div",
+          :confirm   => N_("Do you want to trigger reloading selected servers?"),
```

Comment:
maybe "reloading of the selected servers"? when I read this, the sentence seems like missing something, but I'm not a tech writer :)